test-dids/did:web:did.actor:bob.json

```diff
@@ -0,0 +1,35 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    {
+      "@base": "did:web:did.actor:bob"
```

Comment:
Hmm, so this raises a good question... DID Processors that are JSON-LD aware should probably set `@base` appropriately... that is, one shouldn't have to do the above declaration with a conforming DID processor.